### PR TITLE
Change the conversion because of RNG validation errors

### PR DESCRIPTION
**Pull request by philschatz**
The Story tracking this work: https://github.com/Connexions/baked-pdf/issues/5

These changes are related to https://github.com/Connexions/cnxml/pull/9 and a result of converting a textbook from CNXML->HTML

### CODE REVIEW

**Comment by philschatz** [comment]
The `algebra-intermediate-book` CNXML->HTML conversion now validates

File: rhaptos/cnxmlutils/xsl/test/cite.cnxml.html [hunk 1]
```diff
@@ -28,12 +29,17 @@
       target='_window'
     >CNN.com</a>.
   </p>
-  <p
+  <div
+    class='not-converted-yet'
+  >NOT_CONVERTED_YET: para</div>
```
Comment (philschatz) [inline rhaptos/cnxmlutils/xsl/test/cite.cnxml.html]: This needs to be fixed

Comment (philschatz) [inline rhaptos/cnxmlutils/xsl/test/cite.cnxml.html]: This will need to be fixed. I am not sure why this is not converted.

**Comment by reedstrm** [comment]
@phil  - we need this PR now to fix the sort issue using the xhtml easybake namespace PR. Can we break out the 'non-breaking' version, or do we just merge it, and deal?

**Comment by philschatz** [comment]
Hmm, it _could_ be split but... 

To test, I use the openstax books and convert them to XHTML and then validate against the `.rng` file (to verify that they are valid XHTML). 

By only incorporating some changes, I could not use the `.rng` files for validation. I guess I could write more tests in here... that'll take a day or 2.

**Pull request closed without merging**